returns/primitives/container.py

```diff
@@ -12,16 +13,18 @@
 class BaseContainer(Immutable, metaclass=ABCMeta):
     """Utility class to provide all needed magic methods to the context."""
 
-    __slots__ = ('_inner_value',)
+    __slots__ = ('_inner_value', '_trace')
```

Comment:
Not all containers have `_trace` 

Can we only change this in `Result`?

Reply:
Done!